tests/BLUEPRINT/cad/test_shieldCAD.py

```diff
@@ -40,7 +40,7 @@ def test_thermalshield(self):
         v = ThermalShieldCAD(self.thermal_shield, neutronics=False)
         volume = v.get_properties()["thermal_shield"]["Volume"]
         # This will change if you change the build pattern in TSCAD
-        assert np.isclose(round(volume, 2), 6.02)
+        assert np.isclose(round(volume, 2), 5.96)
```

Comment:
With percentage changes this is now closer (same for vesselCAD) still not the same though

Reply:
Yeah I'm a little concerned these changes are not just butterfly effects. I would have thought that they wouldn't change at all. I have to say though, I don't think it is worth digging down into these (a bit of a mammoth task) because this is for old BLUEPRINT which we're moving away from. They are also not massive so perhaps we can let them slide... I was kind of hoping the change to percent for ripple would fix this.